### DISCUSSION TITLE
savestates in libretro port

### DISCUSSION THREAD

**Discussion by eadmaster** [flyinghead/flycast#1041] [Q&A]
[recent builds](https://buildbot.libretro.com/nightly/windows/x86_64/) seem to have savestates disabled, while they were supported previously:
https://github.com/libretro/flycast/issues/204

Is it possible to enable them back via a source code toggle?

**Comment by flyinghead** [comment]
Save states aren't disabled.
What happens when you try to load or save a state?

**Comment by eadmaster** [comment]
nevermind, it was a problem with my [setup missing access to the info files](https://www.reddit.com/r/RetroArch/comments/yds1aj/missing_save_state_options_on_nintendo_switch/).

For some reason if they are missing the savestate feature gets disabled...
